.github/workflows/codeql-analysis.yml

```
@@ -13,10 +13,10 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ master, v18, v19, v20 ]
+    branches: [ master, v19, v20, v21, make-v22 ]
```

```suggestion
    branches: [ master, v19, v20, v21 ]
```

- remove it before merging
- make v21 branch